components/auth/GoogleSignIn.tsx

```
@@ -12,19 +12,24 @@ export const GoogleSignIn = ({ redirect }: GoogleSignInProps) => {
-      Continue with Google
-    </button>
+    <div className="flex flex-col items-center space-y-4">
+      <button
+        onClick={handleGoogleSignIn}
+        className="bg-white border rounded-md px-6 py-2 shadow-md hover:shadow-lg transition flex items-center"
+      >
+        <img src="/assets/google.svg" alt="Google Logo" className="w-5 h-5 mr-2" />
+        Continue with Google
+      </button>
+      <p className="text-sm font-bold text-gray-500 text-center mt-2">
+        After signing in, you'll be redirected to the home page.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)